python/packages/aws-lambda-sdk/scripts/build-layer-archive.sh

```diff
@@ -1,23 +1,31 @@
 #!/usr/bin/env bash
 set -Eeuo pipefail
-shopt -s extglob globstar
 
-BUILD_DIR=$1       # path to folder where build artifacts are found
-OUTPUT=$2          # where to put the actual artifact
 DIST=$(mktemp -d)  # directory to store intermediate artifacts
 CURRENT_DIR=$(pwd)
 
-mkdir -p $DIST/{python/lib/python3.9/site-packages,sls-sdk-python}
+case $1 in
+  /*) OUTPUT=$1 ;;
+  *) OUTPUT=$CURRENT_DIR/$1 ;;
+esac
+
+SITE_PACKAGES_DIR=python/lib/python3.9/site-packages
+mkdir -p $DIST/{$SITE_PACKAGES_DIR,sls-sdk-python}
+
+INSTALL_DIR=$DIST/$SITE_PACKAGES_DIR
+python3 -m pip install serverless-aws-lambda-sdk --target=$INSTALL_DIR
```

Comment:
@selcukcihan I just started thinking about that, and I think that's incorrect.

This script should build the layer which represents state in repostotyr. It's important as it's used in integration tests, and in integration tests confirm that current state in `main` is working, (and not latest release in PyPI)

Reply:
Yeah I think that makes sense, that integration tests use `main`. Which means we somehow need to build and install the main version of the package before creating the layer. Should we do this within the integration test script or can we push it out of the script like for instance previous step in the CI flow? What do you think?

Reply:
> Which means we somehow need to build and install the main version of the package before creating the layer. 

Integration tests are run in the assumption that all dependencies are installed (integration tests on its own should not pursue deps installation step). It's just build layer step that's integrated in integration tests flow (and that's covered in https://github.com/serverless/console/pull/483).

And in CI we already have ensured that dependencies from PyPI are installed, so if this build script will ensure that layer is build of code in repository, all should work out of a box

Reply:
https://github.com/serverless/console/pull/488 to build the lambda sdk package from code in the repo instead of pypi.